### DISCUSSION TITLE
Versioning/edit of Evaluator Configuration

### DISCUSSION THREAD

**Discussion by filipmihal** [langfuse/langfuse#3798] [Ideas]
### Describe the feature or potential improvement

Hello, it would be great if I could create a new config just by duplicating a recent one and editing the necessary parts. Now, whenever I need to update a config, I have to create a new one from scratch which is quite tedious and time-consuming.
![image](https://github.com/user-attachments/assets/1384447a-6773-4c71-850d-88499fe747ce)

**Comment by marcklingen** [comment]
thanks for sharing this! we plan to make many small UX improvements to the evals feature, this is awesome feedback

**Reply by dveyarangi** [reply]
also edit, delete and reactivate config options would be nice )
